model/labels/labels.go

```diff
@@ -46,9 +47,11 @@ type Label struct {
 // instantiation.
 type Labels []Label
 
-func (ls Labels) Len() int           { return len(ls) }
-func (ls Labels) Swap(i, j int)      { ls[i], ls[j] = ls[j], ls[i] }
-func (ls Labels) Less(i, j int) bool { return ls[i].Name < ls[j].Name }
+func (ls Labels) Len() int      { return len(ls) }
+func (ls Labels) Swap(i, j int) { ls[i], ls[j] = ls[j], ls[i] }
+func (ls Labels) Less(i, j int) int {
```

Comment:
This one strikes me as wrong, as the interface is intended for `sort.Sort` which hasn't changed.